trace_state/HTTP_HEADER_FORMAT.md

```diff
@@ -23,15 +23,15 @@ Maximum length of a combined header MUST be less than 512 bytes.
 
 Url encoded string. Spaces MUST be trimmed from beginning and the end of the name. Names are case sensitive. Vendor libraries encouraged to use namespaced names of the properties to avoid conflicts.
 
-Names `id`, `span-id`, `trace-id`, `sampled` are reserved. These properties are defined in `Trace-Context` header.
+Names `id`, `span-id`, `trace-id`, `sampled` are reserved. These properties are defined in `Trace-Parent` header.
```

Comment:
I think most of the commercial vendors will keep their trace-id, span-id in this part too, just like DynaTrace and AWS suggest.

Reply:
yes. I still think we need to avoid using these names in `Trace-State` to avoid confusions.

Reply:
@SergeyKanzhelev I argue about these names, just because we can't sure the vendors are going to use key=value in vendor-segments, or whether should we suggest to do so. That is based on encoding parts of `trace-state`.